{{ cookiecutter.package_name|slugify }}/{% if cookiecutter.continuous_integration == "GitLab" %}.gitlab-ci.yml{% endif %}

```diff
@@ -127,8 +127,9 @@ Publish:
     - .docker
   stage: deploy
   variables:
+    DOCKER_PUSH: 1
```

Comment:
Could you put this line in alphabetical order under the `variables:` section?

Reply:
Done